packages/fakta-tilrettelegging/src/TilretteleggingFaktaIndex.tsx

```diff
@@ -12,14 +12,15 @@ const intl = createIntl(messages);
 
 interface Props {
   svangerskapspengerTilrettelegging: FodselOgTilrettelegging;
-  arbeidOgInntekt: ArbeidOgInntektsmelding;
+  arbeidOgInntekt?: ArbeidOgInntektsmelding;
   arbeidsgiverOpplysningerPerId: ArbeidsgiverOpplysningerPerId;
   submittable: boolean;
   readonly: boolean;
 }
 
 export const TilretteleggingFaktaIndex = ({ arbeidOgInntekt, ...rest }: Props) => (
-  <RawIntlProvider value={intl}>
-    <TilretteleggingFaktaForm aoiArbeidsforhold={arbeidOgInntekt.arbeidsforhold} {...rest} />
-  </RawIntlProvider>
-);
+    <RawIntlProvider value={intl}>
+      <TilretteleggingFaktaForm aoiArbeidsforhold={arbeidOgInntekt?.arbeidsforhold ?? []} {...rest} />
```

Comment:
Siden vi først er her: siden det kun er arbeidsforhold som brukes, så kan man kanskje kun sende ned det? og ikke hele arbeidOgInntekt?

Reply:
Ser det skaper så mye flere endringer i stories og sånn. Så not worth, disregard